src/main/java/com/baulsupp/oksocial/Main.java

```diff
@@ -501,6 +514,38 @@ public void initialise() throws Exception {
         new Authorisation(serviceInterceptor, credentialsStore, authClient, outputHandler);
 
     clientBuilder.networkInterceptors().add(0, serviceInterceptor);
+
+    if (zipkin) {
+      OkHttpSender sender = OkHttpSender.create("http://localhost:9411/api/v1/spans");
+      AsyncReporter<zipkin.Span> reporter = AsyncReporter.create(sender);
+
+      Tracing tracing = Tracing.newBuilder()
+          .localServiceName("oksocial")
+          .reporter(reporter)
+          .sampler(Sampler.ALWAYS_SAMPLE)
+          .build();
+
+      HttpTracing httpTracing = HttpTracing.create(tracing);
+
+      Tracer tracer = tracing.tracer();
+
+      Consumer<TraceContext> opener =
+          tc -> closeables.add(
+              () -> openLink("http://localhost:9411/zipkin/traces/" + tc.traceIdString()));
```

Comment:
No, happy to spawn a browser.  But my question was really more around 

https://github.com/yschimke/oksocial/pull/255#issuecomment-315769686

I'm wondering about something like

$ cat ~/.zipkinrc
ZIPKIN_SINK=http://localhost:8411/api/v1/trace
ZIPKIN_UI=http://localhost:8411/zipkin/traces/{traceid}

vs

$ cat ~/.zipkinrc
ZIPKIN_SINK=scribe://localhost:1463
ZIPKIN_UI=https://zipkin.company.com/zipkin/traces/{traceid}